src/components/Landing/AuthCard.vue

```diff
@@ -1,17 +1,21 @@
 <template>
   <div class="my-12 shadow-lg w-80 p-4 flex flex-col bg-slate-100 rounded-lg">
     <div v-if="auth.user">
-      <h2 class="font-medium leading-tight text-2xl text-grey-600 mt-2 p-2">
+      <h2
+        class="font-medium leading-tight text-3xl text-grey-600 mt-6 p-4 mb-5"
+      >
```

Comment:
I believe there's too much margin between the h2 and the button and it doesn't looks ok to me 🤓